Anomaly/Defs/Ammo/NerveSpikerBolt.xml

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<ThingCategoryDef>
+		<defName>AmmoNerveSpiker</defName>
+		<label>nerve spiker</label>
+		<parent>AmmoNeolithic</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberArrow</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_NerveSpikerBolt</defName>
+		<label>nerve spiker bolts</label>
+		<ammoTypes>
+			<Ammo_NerveSpikerBolt>Projectile_NerveSpikerBolt</Ammo_NerveSpikerBolt>
+		</ammoTypes>
+	</CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoCrossbowBoltBase">
+		<defName>Ammo_NerveSpikerBolt</defName>
+		<label>nerve spiker bolt</label>
+		<description>A bolt of bioferrite, tipped with biotoxin and shaped to be fired from a nerve spiker.\n\nThough simple to make, these bolts are brittle and tend to deform or break on impact, making them impossible to reuse.</description>		
+		<graphicData>
+			<texPath>Things/Ammo/Medieval/Bolt/Steel</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<statBases>
+			<Mass>0.098</Mass>
+			<Bulk>0.29</Bulk>
+			<Flammability>0</Flammability>
+			<MarketValue>0.78</MarketValue> <!-- The value of 1 bioferrite + value of work [(workAmount / 1200) / product quantity] -->
+		</statBases>
+		<thingCategories>
+			<li>AmmoNerveSpiker</li>
+		</thingCategories>		
+		<tradeTags inherit="false">
+			<li>CE_PreIndustrialAmmo</li>
+		</tradeTags>		
+		<ammoClass>Nerve</ammoClass>
+	</ThingDef>
+
+	<!-- ================== Projectiles ================== -->
+
+	<ThingDef ParentName="BaseCrossbowBoltProjectile">
+		<defName>Projectile_NerveSpikerBolt</defName>
+		<label>nerve spiker bolt</label>
+		<graphicData>
+			<texPath>Things/Projectile/NerveSpikerShot</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>Nerve</damageDef>
+			<speed>34</speed>			
```

Comment:
My revised formula states 28, the same formula currently in use for all arrows and bolts.